src/Microsoft.MobileBlazorBindings.WebView.Windows/WebViewExtendedAnaheimRenderer.cs

```diff
@@ -132,7 +132,7 @@ private void HandleWebResourceRequested(object sender, CoreWebView2WebResourceRe
             var uriString = (string)uriProperty.GetValue(nativeRequest);
             var uri = new Uri(uriString);
 
-            if (Element.SchemeHandlers.TryGetValue(uri.Scheme, out var handler))
+            if (Element.SchemeHandlers.TryGetValue(uri.Scheme, out var handler) && _coreWebView2Environment != null)
```

Comment:
Hmm this one is a bit odd. When/how did you see it null here?

Reply:
When you do "Inspect" and you open the developer tools with Edge. I had no further time to look into it, but I thought that a nullreferencexception was at least something that we do NOT want to have ;-)

Reply:
Ah that's certainly an unusual scenario for the code to handle, but I agree NullRef is bad. I just wonder if there's some other fix that is better long-term but I'm not worried about that right now. This is fine 😄